### PR TITLE
Hot Backups: Restoration required maintenance mode in Active Failover setups

### PR DESCRIPTION
**Pull request by Simran-B**
This is a workaround to avoid non-restored followers becoming leader.

### CODE REVIEW

File: 3.5/programs-arangobackup-limitations.md [hunk 1]
```diff
@@ -42,6 +42,16 @@ dropping of cluster databases, collections and indexes.
 It must be ensured that for the hot backup no such changes are made to the
 cluster's inventory, as this could lead to inconsistent hot backups.
 
+### Active Failover Special Limitations
+
+When restoring hot backups in Active Failover setups, it is necessary to
+prevent that a non-restored follower becomes leader by temporarily setting
+the maintenance mode:
+
+1. `curl -X PUT <leader-endpoint>/_admin/cluster/maintenance -d'"on"'`
```
Comment (KVS85) [inline 3.5/programs-arangobackup-limitations.md]: Is that resilensingle endpoint?

File: 3.5/programs-arangobackup-limitations.md [hunk 2]
```diff
@@ -42,6 +42,16 @@ dropping of cluster databases, collections and indexes.
...
+2. Restore the Hot Backup
+3. `curl -X PUT <leader-endpoint>/_admin/cluster/maintenance -d'"off"'`
+
```
Comment (KVS85) [inline 3.5/programs-arangobackup-limitations.md]: Is that resilensingle endpoint?

File: 3.6/programs-arangobackup-limitations.md [hunk 3]
```diff
@@ -37,6 +37,16 @@ dropping of cluster databases, collections and indexes.
 It must be ensured that for the hot backup no such changes are made to the
 cluster's inventory, as this could lead to inconsistent hot backups.
 
+### Active Failover Special Limitations
+
+When restoring hot backups in Active Failover setups, it is necessary to
+prevent that a non-restored follower becomes leader by temporarily setting
+the maintenance mode:
+
+1. `curl -X PUT <leader-endpoint>/_admin/cluster/maintenance -d'"on"'`
```
Comment (KVS85) [inline 3.6/programs-arangobackup-limitations.md]: Is that resilensingle endpoint?

File: 3.6/programs-arangobackup-limitations.md [hunk 4]
```diff
@@ -37,6 +37,16 @@ dropping of cluster databases, collections and indexes.
...
+2. Restore the Hot Backup
+3. `curl -X PUT <leader-endpoint>/_admin/cluster/maintenance -d'"off"'`
```
Comment (KVS85) [inline 3.6/programs-arangobackup-limitations.md]: Is that resilensingle endpoint?

File: 3.7/programs-arangobackup-limitations.md [hunk 5]
```diff
@@ -37,6 +37,16 @@ dropping of cluster databases, collections and indexes.
 It must be ensured that for the hot backup no such changes are made to the
 cluster's inventory, as this could lead to inconsistent hot backups.
 
+### Active Failover Special Limitations
+
+When restoring hot backups in Active Failover setups, it is necessary to
+prevent that a non-restored follower becomes leader by temporarily setting
+the maintenance mode:
+
+1. `curl -X PUT <leader-endpoint>/_admin/cluster/maintenance -d'"on"'`
```
Comment (KVS85) [inline 3.7/programs-arangobackup-limitations.md]: Is that resilensingle endpoint?

File: 3.7/programs-arangobackup-limitations.md [hunk 6]
```diff
@@ -37,6 +37,16 @@ dropping of cluster databases, collections and indexes.
...
+2. Restore the Hot Backup
+3. `curl -X PUT <leader-endpoint>/_admin/cluster/maintenance -d'"off"'`
```
Comment (KVS85) [inline 3.7/programs-arangobackup-limitations.md]: Is that resilensingle endpoint?

File: 3.5/programs-arangobackup-limitations.md (hunk 1 above)
Comment (Simran-B) [reply 3.5/programs-arangobackup-limitations.md]: @KVS85 Not sure what you mean. You should talk to the leader instance, not a follower. Is the leading instance's endpoint called "resilentsingle"? It's the first time I hear about it.

Comment (KVS85) [reply 3.5/programs-arangobackup-limitations.md]: I meant there are agency and single server (resilentsingle) for each leader/follower node. Is that a call to single server endpoint?

Comment (Simran-B) [reply 3.5/programs-arangobackup-limitations.md]: So the question is whether the leading agent or the leading single server is supposed to be contacted. Given the path I think it is the leading single server, the agency paths are different aren't they?

Comment (maxkernbach) [reply 3.5/programs-arangobackup-limitations.md]: `/_admin/cluster/maintenance` is the endpoint of the **leader DBserver** (leading single server)

It is also possible to enable/disable the maintenance mode via the agency, but the DBserver approach is the easier one. So all that has to be done is rename `<leader-endpoint>` into `<leader-single-endpoint>` or similar.

**Pull request merged**